build.gradle.kts

```diff
@@ -109,11 +111,18 @@ allprojects {
         configFile = rootProject.file("resources/tx-checkstyle-config.xml")
         configDirectory.set(rootProject.file("resources"))
 
+        tasks.checkstyleMain {
```

Comment:
why do we need this? shouldn't `checkstyleMain` run on the `main` source set anyway?

Reply:
I was looking into checking also `resources` file (as `.ServiceExtension` files), but the gradle checkstyle plugin by default is excluding everything not java:
https://github.com/gradle/gradle/blob/f8ee6cae5bd056483bafec72043744540f532e09/platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CheckstylePlugin.java#L139

in any case I forgot to do it :sweat_smile:  will do it right now

Reply:
interesting 🤔 maybe a comment would be useful then?